statsmodels/stats/proportion.py

```diff
@@ -123,6 +123,185 @@ def func(qi):
         raise NotImplementedError('method "%s" is not available' % method)
     return ci_low, ci_upp
 
+def multinomial_proportions_confint(counts, alpha=0.05, method='goodman'):
+    '''Confidence intervals for multinomial proportions.
+
+    Parameters
+    ----------
+    counts : array_like of int, 1-D
+        Number of observations in each category.
+    alpha : float in (0, 1), optional
+        Significance level, defaults to 0.05.
+    method : {'goodman', 'sison_glaz'}, optional
```

Comment:
use dash not underline in options `sison-glaz`  
I think that is more standard, and I had it misspelled when I tried an example